src/index.html

```diff
@@ -17,6 +17,79 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="Moyo logo"
+      />
```

Comment:
[checklist](https://github.com/mate-academy/layout_moyo-header?tab=readme-ov-file#checklist):  Logo is an image wrapped with a link